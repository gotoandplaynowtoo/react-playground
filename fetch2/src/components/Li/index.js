import styled from 'styled-components';

const Li = styled.li`
  padding: 4px 4px;
  margin: 4px 0;
  background: #eee;
`;

export default Li;