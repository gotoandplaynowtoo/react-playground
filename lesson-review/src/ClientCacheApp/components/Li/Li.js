import styled from 'styled-components';
import PropTypes from 'prop-types';

const Li = styled.li`
  background: #aaa;
  margin: 4px 0;
  padding: 4px;
`;

Li.propTypes = {
  children: PropTypes.node
};

Li.defaultProps = {
  children: 'Li place holder'
};

export default Li;