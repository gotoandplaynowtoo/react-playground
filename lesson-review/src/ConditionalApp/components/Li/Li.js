import PropTypes from 'prop-types';
import styled from 'styled-components';

const Li = styled.li`
  background: #aaa;
  padding: 6px;
`;

Li.propTypes = {
  children: PropTypes.node
};

export default Li;