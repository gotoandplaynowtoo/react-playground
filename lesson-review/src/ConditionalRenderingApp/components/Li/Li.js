import styled from 'styled-components';

const Li = styled.li`
  background: #aaa;
`;

export default Li;