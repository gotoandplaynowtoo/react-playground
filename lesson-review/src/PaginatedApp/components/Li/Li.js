import styled from 'styled-components';
import PropTypes from 'prop-types';

const Li = styled.li`
  background: #eee;
  margin: 4px 0;
  padding: 6px;
`;

Li.propTypes = {
  children: PropTypes.node
};

export default Li;