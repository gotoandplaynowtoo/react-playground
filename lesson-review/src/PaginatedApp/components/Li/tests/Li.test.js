import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Li from 'PaginatedApp/components/Li';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Li/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('matches to snapshot', () => {
  const tree = renderer.create(<Li/>).toJSON();
  expect(tree).toMatchSnapshot();
});