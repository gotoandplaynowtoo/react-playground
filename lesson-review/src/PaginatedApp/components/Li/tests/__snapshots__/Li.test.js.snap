// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches to snapshot 1`] = `
<li
  className="sc-bdVaJa gRrJQh"
/>
`;
